### PR TITLE
Bump github.com/mattn/go-isatty

### PR DESCRIPTION
**Pull request by SocalNick**

### CODE REVIEW

**Comment by SocalNick** [comment]
Yay! Only 331 files changes...

**Review by breanneboland — APPROVED** [review]
331 files! 💀 Perfect, looks great. Thanks for taking care of this.

**Comment by SocalNick** [comment]
@breanneboland FYI, I'm holding off on merging this because updating `committer` across all our repos is kind of a pain. We have[ a PR about to go in that will let us manage committer w/ chef](https://github.com/Gusto/cpe-chef/pull/1207), so I'm holding off until that lands.

**Pull request merged**
